trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/create-record-correction-request/index.ts

```diff
@@ -79,6 +79,10 @@ export const postCreateRecordCorrectionRequest = async (req: PostCreateRecordCor
     const formHasErrors = errors.errorSummary.length !== 0;
 
     if (!formHasErrors) {
+      const parsedFormValues = formValues as RecordCorrectionTransientFormData;
```

Comment:
This might be more hassle than it is worth, but can we try and refactor this so the `as` is not required? It's a little risky if the validation function ever fell out of line with this type if we added new fields for example